lib/validate.js

```diff
@@ -58,26 +58,33 @@ async function usernameExists (client, name) {
   return !!user
 }
 
-export function advPostSchemaMembers (client) {
+export function advPostSchemaMembers (client, me) {
   return {
     boost: intValidator
       .min(BOOST_MIN, `must be blank or at least ${BOOST_MIN}`).test({
         name: 'boost',
         test: async boost => !boost || boost % BOOST_MIN === 0,
         message: `must be divisble be ${BOOST_MIN}`
       }),
-    // XXX this lets you forward to youself (it's financially equivalent but it should be disallowed)
     forward: array()
       .max(MAX_FORWARDS, `you can only configure ${MAX_FORWARDS} forward recipients`)
       .of(object().shape({
-        nym: string().required('must specify a stacker').test({
-          name: 'nym',
-          test: async name => {
-            if (!name || !name.length) return true
-            return await usernameExists(client, name)
-          },
-          message: 'stacker does not exist'
-        }),
+        nym: string().required('must specify a stacker')
+          .test({
+            name: 'nym',
+            test: async name => {
+              if (!name || !name.length) return false
```

Comment:
This line also changed, I think this was a bug from before (changed return from `true` to `false`)

Reply:
I think this was there because of the compact call or something ... I remember posts being invalid without forwards or something. I'll double check before merging.